packages/@react-stately/tag/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@react-stately/tag",
+  "version": "3.0.0-alpha.1",
+  "description": "Spectrum UI components in React",
+  "license": "Apache-2.0",
+  "main": "dist/main.js",
+  "module": "dist/module.mjs",
+  "types": "dist/types.d.ts",
+  "exports": {
+    "types": "dist/types.d.ts",
+    "import": "dist/module.mjs",
+    "require": "dist/main.js"
+  },
+  "source": "src/index.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/react-spectrum"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.6.2",
+    "@react-stately/grid": "^3.4.1",
+    "@react-stately/list": "^3.6.0",
+    "@react-stately/utils": "^3.0.0",
+    "@react-types/tag": "3.0.0-beta.0",
+    "react-stately": "^3.19.0"
```

Any reason to have the `react-stately` monopackage as a dep here? Also don't think you need `"@react-stately/grid"` and  `"@react-stately/utils": "^3.0.0",` anymore

```suggestion
    "@react-stately/list": "^3.6.0",
    "@react-types/tag": "3.0.0-beta.0",
```